ksql-engine/src/main/java/io/confluent/ksql/util/SerDeUtil.java

```diff
@@ -94,4 +100,40 @@
     }
   }
 
+  public static synchronized Schema getSchemaFromAvro(String avroSchemaString) {
```

Comment:
package-private

Reply:
Moved them to `KsqlTopicSerDe` as you suggested.